### PR TITLE
add inspect message

### PR DESCRIPTION
**Pull request by madhur-tandon**
Adds the ability to use the `inspect` message protocol in JupyterLite.

### CODE REVIEW

**Comment by madhur-tandon** [comment]
Screenshot 2021-07-12 at 4 18 55 PM

**Comment by jtpio** [comment]
Nice, thanks Madhur!

**Comment by jtpio** [comment]
Looks like there is a small conflict to resolve now that https://github.com/jupyterlite/jupyterlite/pull/239 is in.

Would you mind doing a quick rebase?

File: packages/pyolite-kernel/src/worker.ts [hunk 1]
```diff
@@ -216,6 +216,17 @@ function complete(content: any) {
   return results;
 }
 
+/**
+ * Inspect the code submitted by a user.
+ *
+ * @param content The incoming message with the code to inspect.
+ */
+function inspect(content: any) {
```
Comment (jtpio) [inline packages/pyolite-kernel/src/worker.ts:224]: Looks like we could use a stronger type than `any` here?

And add `content` to the docstring above.

Comment (madhur-tandon) [reply packages/pyolite-kernel/src/worker.ts]: `content` is already in docstring. I have now added type definition.

**Comment by bollwyvl** [comment]
Additionally, as you're closer to it, can you take a pass over the `docs/kernels` matrix and update with the features _actually_ supported? Probably need `comm` as well, now :laughing:. I kinda slammed out the original, and it's probably still kinda wrong. In an ideal world, we'd derive it (or more likely, check it, because no browsers are invited to RTD) from tags in #6, but for now it's all elbow grease...

**Review by jtpio — APPROVED** [review]
Looks good, thanks!

**Comment by jtpio** [comment]
> can you take a pass over the `docs/kernels` matrix and update with the features _actually_ supported?

Yes that would be great :+1:  Opening a new issue to track this.

**Pull request merged**